pages/desktop/user.py

```diff
@@ -117,10 +118,31 @@ def notification_header_text(self):
 
     def click_update_account(self):
         self.selenium.find_element(*self._update_account_locator).click()
-        WebDriverWait(self.selenium, self.timeout).until(lambda s: self.update_message == "Profile Updated")
-
-    def change_hide_email_state(self):
-        self.selenium.find_element(*self._hide_email_checkbox).click()
+        WebDriverWait(self.selenium, self.timeout).until
```

Comment:
I'm not a big fan of the way this is indented (or rather, not indented). I'm actually surprised that this isn't a `pep8` violation, but it is not. I think it's more readable (and more consistent with how we usually write our code) to do it like:

```
        WebDriverWait(self.selenium, self.timeout).until(
            expected_conditions.visibility_of(email_checkbox))
```

The same goes for the other calls to `WebDriverWait` that are lacking this indentation on the 2nd line.